### PR TITLE
Feature: Autoplay Library

### PR DESCRIPTION
**Pull request by pichfl**
Re #218

### CODE REVIEW

File: BookPlayer/Library/BaseListViewController.swift [hunk 1]
```diff
@@ -152,23 +154,57 @@ class BaseListViewController: UIViewController {
...
+        guard
+            shouldAutoplay,
+            let remainingItems = self.items.split(whereSeparator: { $0 == startItem }).last
+            else {
+                return books
+        }
+
+        for item in remainingItems {
+            if let playlist = item as? Playlist, let playlistBooks = playlist.books?.array as? [Book] {
+                for book in playlistBooks {
```
Comment (GianniCarlo) [inline BookPlayer/Library/BaseListViewController.swift]: instead of this `for` loop, you can do `books.append(contentsOf: playlistBooks)`

Comment (pichfl) [reply BookPlayer/Library/BaseListViewController.swift]: Ever learning. Will update.

File: BookPlayer/Library/LibraryViewController.swift [hunk 2]
```diff
@@ -302,22 +302,30 @@ extension LibraryViewController {
...
+        if let book = self.items[indexPath.row] as? Book {
+            let books = self.queueBooksForPlayback(book)
 
-                self.navigationController?.pushViewController(playlistVC, animated: true)
-            }
+            self.setupPlayer(books: books)
+        }
+    }
 
+    private func presentPlaylist(_ playlist: Playlist) {
```
Comment (GianniCarlo) [inline BookPlayer/Library/LibraryViewController.swift]: testing webhooks in pr comments

**Comment by GianniCarlo** [comment]
There's a inconsistency between the lock screen player and the current book playing 
img_1669
img_1670

**Comment by GianniCarlo** [comment]
If the playthrough starts with a playlist (via artwork tap) the autoplay won't work

**Comment by pichfl** [comment]
I think you may have discovered another bug with how changed information is sent to `MPNowPlayingInfoCenter`. I'll keep that separate from this PR but will look into it.

**Comment by pichfl** [comment]
Before we merge this I we should discuss what autoplay actually does. Should it skip completed files? It does when starting a playlist.

**Comment by GianniCarlo** [comment]
👌 sorry for the delay. For documentation/traceability sake: as we discussed (and already implemented in this PR) completed files will be skipped

**Comment by pichfl** [comment]
This is also noted in the help text on the settings screen, see here:
https://github.com/TortugaPower/BookPlayer/blob/8742bd956dafed5a9510452c096e5ea5021ae9f3/BookPlayer/Base.lproj/Main.storyboard#L88

**Pull request merged**